src/main/java/com/google/devtools/build/lib/syntax/ParamDescriptor.java

```diff
@@ -54,8 +53,7 @@ private ParamDescriptor(
     this.allowedTypes = allowedTypes;
```

Comment:
side-note (unrelated to this PR): arguments `callbackEnabled` and `doc` are unused